src/stores/compliance-ccpa-guide.md

```diff
@@ -0,0 +1,145 @@
+
+    **Request to Know**: A request that you disclose the personal information that you have collected about the customer, and includes the following:
+
+    - Specific pieces of personal information that you have collected about the customer
+    - Categories of personal information that you have collected about the customer
+    - Categories of sources from which the personal information is collected
+    - Categories of personal information about the customer that you have sold or disclosed for a business purpose
+    - Categories of third parties to whom the personal information was sold or disclosed for a business purpose
+    - The reasons why your business collects and/or sells personal information.
+
+### Step 6: Review your agreements with service providers.
+
+**Stakeholders:** Legal, Administrative Support
+   
+Review and update all service provider contracts to reflect CCPA requirements.
+
+### Step 7: Update your privacy policy.
+
+**Stakeholders:** Legal, Administrative Support
+
+Review your current privacy policy and consider what, if any, additional disclosures are necessary. Your privacy policy should describe your online and offline business practices regarding the collection, use, disclosure, and sale of personal information and the rights of consumers to determine how their information is used. To learn more, see the [California Online Privacy Protection Policy][1] (CalOPPA)  
+
+**Use of Personal Information**: You must disclose what personal information is collected, as well as any financial incentives you receive in exchange from the sale of personal information. You must explain how the incentive is allowed under CCPA, and how the value of the personal information is calculated.
+   
+**Age of Consent**: Businesses that exclusively target consumers under 16 years of age and do not sell their personal information without permission are not required to provide an opt-out notice. Others must follow these guidelines: 
+
+- **Minors < 13**: Parental authorization is required for minors under the age of 13 to opt-in to the sale of their personal information.
+
+- **Minors 13 to < 16**: Minors at least 13 and less than 16 years of age can opt-in to the sale of their personal information, provided the business establishes a reasonable process to document the action. The process must be set forth in the company's [privacy policy]({% link stores/privacy-policy.md -%}). When a business receives requests from minors in this age range, it must inform them of their right to opt-out at a later date, and explain how to do it.
+
+### Step 8: Document all CCPA-related procedures and maintain records.
+
+**Stakeholders:** Customer Service, Administrative Support
+
+For a period of 24 months after each request received, maintain a record of the request and your company's response. 
+
+[1]: https://en.wikipedia.org/wiki/Online_Privacy_Protection_Act 
+[2]: https://devdocs.magento.com/guides/v2.3/architecture/gdpr/magento-2x.html
```

Comment:
The link for GDPR topics in devdocs will most likely change when you start working on the devdocs updates. Redirects will probably take care of it int he short-term, but we'll need to make a note to come back and update these links later.

Reply:
All the links should be in sync now.